docs/source/private/overview.rst

```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
 You can open and edit the file using e.g. VIM or Microsoft Word. Please fill out this form for each private dataset and upload
 the file (as a pdf or text file) together with the other results in your zenodo upload.
 
+.. _private-data-gather:
 
-How will this be gathered?
-==========================
+Uploading the results to zenodo
+===============================
 
-.. _private-data-gather:
+To prepare the results for the submission, please follow these steps:
 
-We anticipating gathering the data through:
+1. Run the :ref:`post-simulation cleanup` script.
+2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
+   The contents in this folder are all human readable and should be reviewed before submitting.
+3. Prepare a CSV file containing the experimental data for the set (see :ref:`here <exp-csv-file>`).
+4. Fill out the :ref:`private dataset info form <private info form>`.
+5. Create a folder named ``dataset_name`` where `dataset_name` can be any arbitrary name that you want to give the different private datasets
```

Comment:
Is the second dataset_name supposed to be highlighted it might be missing the second set of `

Reply:
```suggestion
5. Create a folder named ``dataset_name`` where ``dataset_name`` can be any arbitrary name that you want to give the different private datasets
```
Just to make it easier to merge in.